content/en/docs/concepts/configuration/secret.md

```diff
@@ -107,16 +167,16 @@ Kubernetes provides several built-in types for some common usage scenarios.
 These types vary in terms of the validations performed and the constraints
 Kubernetes imposes on them.
 
-| Built-in Type                         | Usage                                   |
-| ------------------------------------- | --------------------------------------- |
-| `Opaque`                              | arbitrary user-defined data             |
-| `kubernetes.io/service-account-token` | ServiceAccount token                    |
-| `kubernetes.io/dockercfg`             | serialized `~/.dockercfg` file          |
-| `kubernetes.io/dockerconfigjson`      | serialized `~/.docker/config.json` file |
-| `kubernetes.io/basic-auth`            | credentials for basic authentication    |
-| `kubernetes.io/ssh-auth`              | credentials for SSH authentication      |
-| `kubernetes.io/tls`                   | data for a TLS client or server         |
-| `bootstrap.kubernetes.io/token`       | bootstrap token data                    |
+| Built-in Type | Usage |
+|--------------|-------|
+| `Opaque`     |  arbitrary user-defined data |
+| `kubernetes.io/service-account-token` | ServiceAccount token |
+| `kubernetes.io/dockercfg` | serialized `~/.dockercfg` file |
+| `kubernetes.io/dockerconfigjson` | serialized `~/.docker/config.json` file |
+| `kubernetes.io/basic-auth` | credentials for basic authentication |
+| `kubernetes.io/ssh-auth` | credentials for SSH authentication |
+| `kubernetes.io/tls` | data for a TLS client or server |
+| `bootstrap.kubernetes.io/token` | bootstrap token data |
```

Comment:
I think I prefer the original formatting here. Open to being convinced though.

Reply:
I reverted it, my rebase was icky